src/main/java/net/dv8tion/jda/api/hooks/ListenerAdapter.java

```diff
@@ -171,6 +171,7 @@ public void onUserUpdateActivityOrder(@Nonnull UserUpdateActivityOrderEvent even
     public void onUserTyping(@Nonnull UserTypingEvent event) {}
     public void onUserActivityStart(@Nonnull UserActivityStartEvent event) {}
     public void onUserActivityEnd(@Nonnull UserActivityEndEvent event) {}
+    public void onUserUpdateFlags(@Nonnull UserUpdateFlagsEvent event) {}
```

Comment:
This should be grouped with the other update events above

Reply:
I don't quite understand what you mean exactly with that.

Reply:
All the other `onUserUpdate` methods are grouped together a few lines above, this one should be in that group.

Reply:
Updated